diff --git a/CMakeLists.txt b/CMakeLists.txt
index d255ead..b00c964 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -96,4 +96,6 @@ endif (WIN32)

 if (UNIX)
     target_link_libraries(ImHex libglfw.so libmagic.so libcrypto.so libdl.so libcapstone.so ${llvm_libs} ${Python_LIBRARIES} nlohmann_json::nlohmann_json)
-endif (UNIX)
\ No newline at end of file
+endif (UNIX)
+
+install(TARGETS ImHex DESTINATION bin)
--
